models/demos/deepseek_v3/tests/test_mlp_1d.py

```diff
@@ -81,7 +81,7 @@ def run_weight_conversion_test(MLPClass, hf_config, state_dict, tmp_path, refere
     # assert Path(weight_config["w3"]["input_tensor_b"]).exists()
 
     # Load and verify a weight
-    w1_ttnn = load_weight(weight_config["w1"]["input_tensor_b"], device=mesh_device)
+    w1_ttnn = ttnn.load_tensor(weight_config["w1"]["input_tensor_b"]).to(device=mesh_device)
```

Comment:
Why do we not load the tensor directly onto the device?

Reply:
Also, we should definitely keep using the  `load_weight` function. Why was this changed?